### DISCUSSION TITLE
git naar laatste release

### DISCUSSION THREAD

**Discussion by vincedh** [dsmrreader/dsmr-reader#1496] [Questions & answers / How-to's] [answered]
Hoi Dennis (of iemand anders),

Waarschijnlijk een makkelijke vraag maar ik ben ooit teruggegaan naar een eerdere versie met
git checkout tags/v[versienummer]

maar het is me niet helemaal duidelijk hoe ik nou weer op de laatste release kom en ook in de toekomst weer met de laatste versies mee loop.

ik hoor het graag & bedankt alvast!

**Comment by dennissiemensma** [comment, marked as answer]
Je kunt "sudo su - dsmr" doen, daarna "git checkout v4" (als je op v4.x zit nu) en daarna "./deploy.sh".

**Comment by vincedh** [comment]
done, thanks!
